sdf/1.10/joint.sdf

```diff
@@ -74,16 +74,16 @@
     </element> <!-- End Dynamics -->
     <element name="limit" required="1">
       <description>specifies the limits of this joint</description>
-      <element name="lower" type="double" default="-1e16" required="1">
+      <element name="lower" type="double" default="-inf" required="1">
         <description>Specifies the lower joint limit (radians for revolute joints, meters for prismatic joints). Omit if joint is continuous.</description>
       </element>
-      <element name="upper" type="double" default="1e16" required="1">
+      <element name="upper" type="double" default="inf" required="1">
         <description>Specifies the upper joint limit (radians for revolute joints, meters for prismatic joints). Omit if joint is continuous.</description>
       </element>
-      <element name="effort" type="double" default="-1" required="0">
+      <element name="effort" type="double" default="inf" required="0">
```

Comment:
The description says "Limit is not enforced if value is negative". Something to 
keep in mind when updating gz-physics.

Reply:
this was already part of the behavior of the DOM object as of https://github.com/gazebosim/sdformat/pull/357